axon-cosmosdb/src/main/java/org/axonframework/extensions/cosmosdb/eventsourcing/tokenstore/CosmosTokenItem.java

```diff
@@ -215,7 +251,9 @@ void setTimestamp(Instant timestamp) {
     @Override
```

Comment:
Why?

Reply:
Because I have made similar "we don't use equals/hashcode" assumptions in the past with Axon Framework + Extensions, that proved to be wrong for the internals of AF, causing a shitty bug hunt.

Next to this, although this PR's code, and/or AF and the extensions _might_ not use this, the `TrackingToken` is shared with our users, injected in events, and potentially in event handlers.
So, next to being careful for ourselves, I figure it makes sense to provide clean POJOs with an expected set of default methods, of which I would definitely assume equals, hashcode, and toString to be a part.

Reply:
Or, perhaps we should make this class `private`. 
Then you're pointer of it is never compared will be true and protectable by ourselves at least.